src/test/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommandTest.java

```diff
@@ -83,6 +116,30 @@ public void writeFiles()
         }
     }
 
+    /**
+     * Similar to runAndTest, however this function will return a {@link TestMapRouletteConnection}
+     *
+     * @param additionalArguments
+     *            String[] of extra arguments for {@link MapRouletteUploadCommand}, to add to the
+     *            data i/o locations and server config
+     * @return TestMapRouletteConnection
+     */
+    private TestMapRouletteConnection run(final String[] additionalArguments)
```

Comment:
runAndTest could call this method to cut down on some duplication

Reply:
Yup, good call. This was handled in the latest commit